packages/postcss/config/stylelint.config.js

```diff
@@ -51,9 +51,6 @@ module.exports = {
     'selector-combinator-space-before': 'always',
     'selector-descendant-combinator-no-non-space': true,
     'selector-list-comma-newline-after': 'always',
-    'selector-max-specificity': ['0,2,1', {
```

Comment:
Do we need to update a sane default for this? Are we running into specificity issues where we're writing too specific styles? Curious to know more about this change.